spec/index.bs

```diff
@@ -6691,7 +6726,337 @@ An <dfn dfn>Extent3D</dfn> is a {{GPUExtent3D}}.
+
+The {{GPUTextureUsage/STORAGE|GPUTextureUsage.STORAGE}} column specifies the support for {{GPUTextureUsage/STORAGE}} usage in the core API, including both {{GPUBindingType/readonly-storage-texture}} and {{GPUBindingType/writeonly-storage-texture}}. If there is a &star; mark, the atomic operations on the storage are also supported for this format.
+
+The "Filter" column specifies whether textures of this format can be sampled in shaders with a sampler having {{GPUFilterMode/linear}} filtering.
+
+<table class='data'>
+    <thead class=stickyheader>
+        <tr>
+            <th>Format
+            <th>{{GPUTextureUsage/OUTPUT_ATTACHMENT}}
+            <th>{{GPUTextureUsage/STORAGE}}
+            <th>Filter
+    </thead>
+    <tr><th class=stickyheader>8-bit per component<th><th><th>
+    <tr>
+        <td data-componenttype=norm>{{GPUTextureFormat/r8unorm}}
+        <td>&checkmark;
+        <td>
+        <td>&checkmark;
+    <tr>
+        <td data-componenttype=norm>{{GPUTextureFormat/r8snorm}}
+        <td>
+        <td>
+        <td>&checkmark;
+    <tr>
+        <td data-componenttype=int>{{GPUTextureFormat/r8uint}}
+        <td>&checkmark;
+        <td>
+        <td>
+    <tr>
+        <td data-componenttype=int>{{GPUTextureFormat/r8sint}}
+        <td>&checkmark;
+        <td>
+        <td>
+    <tr>
+        <td data-componenttype=norm>{{GPUTextureFormat/rg8unorm}}
+        <td>&checkmark;
+        <td>
+        <td>&checkmark;
+    <tr>
+        <td data-componenttype=norm>{{GPUTextureFormat/rg8snorm}}
+        <td>
+        <td>
+        <td>&checkmark;
+    <tr>
+        <td data-componenttype=int>{{GPUTextureFormat/rg8uint}}
+        <td>&checkmark;
+        <td>
+        <td>
+    <tr>
+        <td data-componenttype=int>{{GPUTextureFormat/rg8sint}}
+        <td>&checkmark;
+        <td>
+        <td>
+    <tr>
+        <td data-componenttype=norm>{{GPUTextureFormat/rgba8unorm}}
+        <td>&checkmark;
+        <td>&checkmark;
+        <td>&checkmark;
+    <tr>
+        <td data-componenttype=norm>{{GPUTextureFormat/rgba8unorm-srgb}}
+        <td>&checkmark;
+        <td>
+        <td>&checkmark;
+    <tr>
+        <td data-componenttype=norm>{{GPUTextureFormat/rgba8snorm}}
+        <td>
+        <td>&checkmark;
+        <td>&checkmark;
+    <tr>
+        <td data-componenttype=int>{{GPUTextureFormat/rgba8uint}}
+        <td>&checkmark;
+        <td>&checkmark;
+        <td>
+    <tr>
+        <td data-componenttype=int>{{GPUTextureFormat/rgba8sint}}
+        <td>&checkmark;
+        <td>&checkmark;
+        <td>
+    <tr>
+        <td data-componenttype=norm>{{GPUTextureFormat/bgra8unorm}}
+        <td>&checkmark;
+        <td>
+        <td>&checkmark;
+    <tr>
+        <td data-componenttype=norm>{{GPUTextureFormat/bgra8unorm-srgb}}
+        <td>&checkmark;
+        <td>
+        <td>&checkmark;
+    <tr><th class=stickyheader><th><th><th>
```

Comment:
Is there meant to be text here?

Reply:
we need some kind of separation, but I couldn't find the right word to describe these two formats. Any ideas? :)

Reply:
"32-bit packed formats"
"variable component widths"
"other component widths"